### DISCUSSION TITLE
[4.x] Workflow Notifications enabled

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#47078] [3. Approved Features]
### Steps to reproduce the issue

- Steps to reproduce the issue
- Enable workflows in the content options with Basic as the default
- Create another super user
- Observe that all the Transitions have notifications enabled
- Create a new article and save & close
- Edit that article and change the stage

### Expected result
Notification email sent

### Actual result
No notifications sent 

### Additional comments
No notifications are sent because the default settings do not have any recipents.
The ability to have a transition with notification enabled but no recipents should not be possible.
Either the fields should be required or have a default value
As it is a user might reasonably assume that the notifications would be sent to every user who has "receive system emails" enabled on their account.

![image](https://user-images.githubusercontent.com/1296369/166877868-3a4d4067-4b18-48b2-acfd-e8e8dde4f120.png)

While it is true that there is no message saying that notifications have been sent you would not know that there should have been a message on a default install

# Approval Information
Approved: 11.03.2026
Minimum Target Version: 6.2

**Comment by HLeithner** [comment]
**The Joomla Maintainer Team decided at the 11.03.2026 meeting to accept this feature in a different way. Doing the same as in the autoupdate notification.**
